README.md

```diff
@@ -1 +1,107 @@
-# Fableous
+# :art: Fableous
+
+[![GitHub Actions Status](https://github.com/deco-finter/fableous/actions/workflows/build.yml/badge.svg)](https://github.com/deco-finter/fableous/actions/workflows/build.yml)
+
+## Service
+
+The application is divided into two parts:
+
+|   Name    |  Code Name  | Stack                             |
+| :-------: | :---------: | --------------------------------- |
+| Back-end  | fableous-be | Go, Gin + Gorm, PostgreSQL, Redis |
+| Front-end | fableous-fe | TypeScript, React                 |
+
+## Develop
+
+### fableous-be
+
+`fableous-be` uses [Go Modules](https://blog.golang.org/using-go-modules) module/dependency manager, hence at least Go 1.11 is required. To ease development, [comstrek/air](https://github.com/cosmtrek/air) is used to live-reload the application. Install the tool as documented.
+
+To begin developing, simply enter the sub-directory and run the development server:
+
+```shell
+$ cd fableous-be
+$ go mod tidy
+$ air
+```
+
+### fableous-fe
+
+To begin developing, simply enter the sub-directory and run the development server:
+
+```shell
+$ cd fableous-fe
+$ npm install

```

Comment:
in init frontend branch, frontend details in readme should be rechecked (I'll do it before PR)

Reply:
I think the only change needed is to use yarn instead of npm here